includes/widgets/ally.php

```diff
@@ -0,0 +1,292 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+use Elementor\Core\Utils\Hints;
+use Elementor\Modules\ContentSanitizer\Interfaces\Sanitizable;
+
+/**
+ * Elementor heading widget.
+ *
+ * Elementor widget that displays an eye-catching headlines.
+ *
+ * @since 1.0.0
```

Comment:
All the `since` tags in this file should be changed to Elementor version, either 3.30.0 or to 3.31.0...

Reply:
removed since I have no Idea what version it will be

Reply:
3.31.0